aws.yaml.example

```diff
@@ -10,6 +10,12 @@ service_notifications_sensor:
   port: 12345
   path: /my-path
 
+action:
```

Comment:
Why is `action` the top-level key?

Reply:
I assumed actions should be treated just like the two sensors (SQS and SNS), being top-level keys.

Reply:
If the roles can potentially be reused by the sensors, I would simply make a top-level `roles` key.

The type of data (ARN) isn't really relevant to anybody using the configuration key, so I would keep the value type out of the key name. Including the ARN in the parameter description is sufficient in my opinion.

If the roles cannot potentially ever be reused by the sensors, keeping the `roles` key underneath a top-level `actions` (plural) key would be fine. I hope all of that makes sense.

Reply:
Totally agree with removing the "_arn" suffix from "roles_arn" field, I just wanted to make it as clear as possible but you are right considering that one should read the description first.

Regarding the use of roles, I assumed that people might want to have separate roles for actions and sensors (see my other PR - #87 - opened some weeks ago). In my case, I use the same roles (and as you said, probably anybody would do that) but I considered to be more specific. If you say it is enough to have only one list of roles, for my use case at least, it fits perfectly.

Reply:
> If you say it is enough to have only one list of roles, for my use case at least, it fits perfectly.

I'm not 100% sure either way. Leave it as it is, since users can still specify the same ARNs for both actions and sensors if they wish. If somebody wants to use a single `roles` key, they can tweak this pack to pull from both the top-level `roles` and the `actions.roles` or the `sensors.roles` keys as appropriate.